transports/msmq/routing_content_msmqtransport_[1,).partial.md

```diff
@@ -1,5 +1,9 @@
 Mapping the logical destination to the physical address containing the queue and host names is the responsibility of physical routing. 
 
+
```

Comment:
Notpick but if the note is the same in all 3 places we should pull it into an "include"

https://github.com/Particular/docs.particular.net#markdown-includes